src/constants.js

```diff
@@ -56,7 +56,8 @@ export const LinearToneMapping = 1;
 export const ReinhardToneMapping = 2;
 export const CineonToneMapping = 3;
 export const ACESFilmicToneMapping = 4;
-export const CustomToneMapping = 5;
+export const AgXToneMapping = 5;
+export const CustomToneMapping = 6;
```

Comment:
Perhaps we could bump CustomToneMapping farther out of this range (99?) so it doesn't change with future additions.

Reply:
Keeping as-is - see https://github.com/mrdoob/three.js/pull/27366#discussion_r1424115499